### ISSUE TITLE
How to layout the results page with a Wrap in a grid view builder

### ISSUE THREAD

**Issue by patkellydesigns** [patkellydesigns/trump_machine#7]
This might be a better solution for the search results UI

**Comment by ThalapathySiva** [comment]
shall I work on this could you explain me clearly...?

**Comment by ThalapathySiva** [comment]
You want to show the search_result page in grid view ?

**Comment by patkellydesigns** [comment]
@ThalapathySiva I played with a GrivView and a SliverGridView and it didn't look nice so I reverted to a ListView. 

The issue was the text within Topic Chips was not a consistent size so some Rows had 1 Chip and some Rows had 3 it looks very ugly.

I haven't spend a lot of time using GridViews so I must have been missing something. I was looking for a solution similar to this widget. [Wrap Widget](https://api.flutter.dev/flutter/widgets/Wrap-class.html)

Any ideas about how to make the Topics list more compact and still look nice? Currently, I feel like scrolling the list one topic at a time is cumbersome.  

I always liked the look of these 

[Tag Cloud ](https://camo.githubusercontent.com/285bdbe252912c305d1117f3e80ec8202c29dfc9/687474703a2f2f6661726d372e7374617469632e666c69636b722e636f6d2f363032332f353934343232383033355f336264353765386462325f622e6a7067)
